README.md

```diff
@@ -295,9 +285,44 @@ Similarly, you can also write sequence schemas that expect particular values in
 (s/validate StringListOrKeywordMap [:foo])
 ;; RuntimeException:  Value does not match schema: [(not (instance? java.lang.String :foo))]
 
+;; if (shorthand for conditional)
+(def StringListOrKeywordMap (s/if map? {s/Keyword s/Keyword} [String]))
+
+;; cond-pre (experimental), also shorthand for conditional, allows you to skip the
+;; predicate when the options are superficially different
+(def StringListOrKeywordMap (s/cond-pre {s/Keyword s/Keyword} [String]))
+;; but don't do this (use `if` or `abstract-map-schema` here instead):
```

Comment:
:+1: 

maybe "`cond-pre` does a greedy match on the 'precondition' of the args, so if two schemas are structurally the same, then the checker will pick the first. For example, the following `BadSchema` will never validate `{:b :x}` because the first arg always wins"


(or maybe that's too wordy?)